### ISSUE TITLE
I can't install cython on mac blender2.78

### ISSUE THREAD

**Issue by mustafamustika** [JacquesLucke/animation_nodes#680]
Hi,
Firstly It's look amazing. Thank you for this awasom addon.

my problem is the same with someone. i have tried everything but it's still doesn't work... 

i can install master version but i need new version :) thank you

![screen shot 2017-04-04 at 1 19 48 pm](https://cloud.githubusercontent.com/assets/26897065/24652291/f4fca852-1939-11e7-9049-01387036fd24.png)

**Comment by JacquesLucke** [comment]
Did you uninstall all other versions of Animation Nodes? If you are unsure checkout the addons folder of Blender. There should only be one folder called "animation_nodes". All other installations have to be removed.

The exact path to the addon directory is in the error message you posted (the second one).

**Comment by mustafamustika** [comment]
Yes i did. Even i uninstall, reinstall blender. Master version can install easily and working correctly.
Can you able to do same thing for cython ? or what can i do I really don't know. I'm trying since morning till now... I'm sure lot of people has same problem with me as I seen on forums.

Thank you for quick answer.

**Comment by JacquesLucke** [comment]
are you downloading the cython branch from github or from graphicall.org ?
Because you have to download it from graphicall.
Can you send me a link to where you downloaded it?

Also if it still does not work, can you send me a screenshot of the addon folder?

**Comment by mustafamustika** [comment]
Hi Jacques,
Finally I did. I have tried many cython and finally it's working. But I have one more problem...

When I connect Objects Group node to Matrix Output then all objects coming in center. After I use Initial Transform... but it's still same. it doesn't work like your tutorial. You think what is the wrong ?
Thank you.

![screen shot 2017-04-05 at 9 16 39 am](https://cloud.githubusercontent.com/assets/26897065/24692295/f372131c-19e0-11e7-9cc5-6fc91147c189.png)

![screen shot 2017-04-05 at 9 16 49 am](https://cloud.githubusercontent.com/assets/26897065/24692305/fc6cc25a-19e0-11e7-99a9-09eed881c89f.png)

![screen shot 2017-04-05 at 9 17 14 am](https://cloud.githubusercontent.com/assets/26897065/24692307/007494a4-19e1-11e7-8f48-39a0fd8c753d.png)
here is the screen capture.
